_data/hosting_providers.json

```diff
@@ -2268,5 +2268,15 @@
     "plan": "",
     "reviewed": "2020.01.07",
     "note": "Available for all plans from free account to dedicated servers"
+  },
+  {
+    "name": "Nuclear.Hosting",
+    "link": "https://nuclear.hosting",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "",
```

Comment:
Any chance you have an announcement page that we can link users to that states that there are no additional steps required to set up an SSL certificate when creating a server? If not, could you please confirm here that this is the case?

Reply:
I got no announcement page or mention about that but I can confirm it. Activation is made by checking the checkbox in the hosting control panel without any further steps.

Reply:
Do you have a site showing where the checkbox is located, so users can be sure to check it? A link to that would go in the "tutorial" field. Based on this description (requiring user action, even if it's a single checkbox), it sounds like nuclear hosting falls under our "partial" support category.

Reply:
Here is some information about LE support on NH: https://nuclear.hosting/help/https/#https-secured-encrypted-access-to-website

Reply:
Perfect. Based on this information, we need to change "full" to "partial", and add this link to the "tutorial" field.

If you ever decide to change it so that that checkbox is checked by default, we can update this.